content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -110,6 +110,9 @@ The output is similar to this:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"v1","kind":"ServiceAccount","metadata":{"annotations":{},"name":"build-robot","namespace":"default"}  
```

Comment:
drop this bit, it only applies to client-side kubectl apply, and is not relevant to the example

Reply:
Thanks, @liggittfor taking a look at this.
I will remove it.

Reply:
@liggitt  Addressed the comment! PTAL